src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -1468,6 +1472,15 @@ class HandshakeForDeflation : public HandshakeClosure {
   }
 };
 
+static size_t delete_monitors(GrowableArray<ObjectMonitor*> delete_list) {
```

Comment:
The argument is passed by value, so is copied.

Reply:
Good catch! I'll fix that.